internal/backup.go

```diff
@@ -373,7 +373,11 @@ func GetBackupSentinelObjects(folder storage.Folder) ([]storage.Object, error) {
 // TODO : unit tests
 // GetBackups receives backup descriptions and sorts them by time
 func GetBackups(folder storage.Folder) (backups []BackupTime, err error) {
```

Comment:
I would add an argument to the old functions, otherwise you need to know the default behavior, but maybe it's ok :)